sale_order_line_date/README.rst

```diff
@@ -14,12 +14,12 @@ of the lines change to match.
 Usage
 =====
 
-Create a Quotation or a Sales Order and fill the requested date in the sale
+Create a Quotation or a Sales Order and it fills the requested date in the sale
```

Comment:
@lreficent I just checked again and the requested date is passed from the sale order to the lines. I think it's working. I'd not modify anything